docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -1,43 +0,0 @@
----
```

Comment:
Are no updates in the redirection file by design?

Reply:
No, thanks for the reminder.